packages/core/src/__tests__/error.test.ts

```diff
@@ -0,0 +1,56 @@
+import {ComponentError, logIfComponentError} from "../utils/ComponentError";
+
+const mockLogHandledError = jest.fn();
+
+jest.mock("react-native", () => ({
+  NativeModules: {
+    EmbraceManager: {
+      logHandledError: (
+        message: string,
+        componentStack: string,
+        params: object,
+      ) => {
+        mockLogHandledError(message, componentStack, params);
+        return true;
```

Comment:
technically the real implementation would be returning something like `Promise.resolve(true)`